frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditTable/EditTableColumns.tsx

```diff
@@ -81,10 +82,11 @@ export const EditTableColumns = ({
             isReadOnly={isLoading}
             isInvalid={!!errors?.columns?.[index]?.title}
           >
-            <Stack justify="space-between" align="center" direction="row">
+            <Grid templateColumns="1fr auto">
```

Comment:
not sure if the padding will be off like this, since stack used to give everything inside 0.5rem of spacing

Reply:
<img width="1505" alt="image" src="https://user-images.githubusercontent.com/25571626/193546182-d9bf1b50-43fd-4ac8-995f-295d9084d920.png">

Looks right, the changes to UI are because the bottomline for the delete button was lower than that for the "Column 1" form labels in the past when it wasn't supposed to be. Now it's aligned, so everything is moved up